app/services/idv/steps/ipp/state_id_step.rb

```diff
@@ -8,19 +8,47 @@ def call
           Idv::StateIdForm::ATTRIBUTES.each do |attr|
             flow_session[:pii_from_user][attr] = flow_params[attr]
           end
+
+          # Accept Date of Birth from both memorable date and input date components
+          fp = flow_params&.[](:dob)
```

Comment:
Can we move this to a helper method or class, and add test coverage for it?

Reply:
Moved and tested.